numpy/fft/fftpack.py

```diff
@@ -69,13 +69,13 @@ def _raw_fft(a, n=None, axis=-1, init_function=fftpack.cffti,
         if s[axis] > n:
             index = [slice(None)]*len(s)
```

Comment:
Maybe nicer to just fix the creation of the index:
```
index = (slice(None),) * axis + (slice(0, n),)
```
then, no need to use `tuple` below.